internal/utils/utils.go

```diff
@@ -117,3 +120,37 @@ func IssueJWT(t *testing.T, cfg JWTConfig) (jwtText, hmacKey string) {
 	}
 	return
 }
+
+// BuildJWTConfig creates a stub for a generic JWT given an existing database and
+// exposure keys
+func BuildJWTConfig(tb testing.TB, db *database.DB, keys []verifyapi.ExposureKey) *JWTConfig {
```

Comment:
update publish_test to use this

Reply:
Still working on getting this working with `iss` conflicts. Got side tracked with a high prio fix 👍

Reply:
Fixed iss, was init db in two places. Now hitting 401